.travis.yml

```diff
@@ -21,6 +21,10 @@ before_script:
 script:
   - yarn lint
   - AST_COMPARE=1 yarn test -- --runInBand
+  # A basic test to ensure prettier runs without devDependencies
+  - rm -rf node_modules
```

Comment:
Hmm, I can't seem to get this to fail when a dependency is only in `devDependencies`. I'll close this for now, since https://github.com/prettier/prettier/pull/2387 is merged.